components/disposable-focus/index.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Re: Folder placement. This is technically not a component, but rather a function which returns a modified component. Dunno if or how we'd want to distinguish this.